.kodiak.toml

```diff
@@ -0,0 +1,13 @@
+version = 1
+
+[merge]
+automerge_label = "🚀 merge it!"
+blocking_labels = ["Status: WIP", "Status: Do Not Merge"]
```

Comment:
What would be a situation in which to use `WIP` and not `Do Not Merge`?

Reply:
🤷🏼 Just giving you options I guess.